app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,10 @@
+class CommentsController < ApplicationController
+  def create
+    @user = User.find(params[:user_id])
```

Comment:
- [ ] You should also create comments on behalf of the current_user as requested in the project requirements.

> Create a comment on behalf of the current_user.

```ruby
    @user = current_user
```